liblangutil/Scanner.h

```diff
@@ -139,12 +164,19 @@ class Scanner
 	///@}
 
 private:
+	inline Token setError(ScannerError _error) noexcept
```

Comment:
I think this should just set the error and not return a token.

Reply:
I cannot really come up with a better name. The fact that a function called `setError` returns something should prompt the reader to look up the function. A function called `setAndReturn` might hint at the fact that it just returns its argument, which is wrong. `setErrorAndReturnIllegal` migth be a fitting name, but is too long, in my opinion.